Sources/BraintreeShopperInsights/BTEligiblePaymentsRequest.swift

```diff
@@ -5,7 +5,7 @@ import PassKit
 struct BTEligiblePaymentsRequest: Encodable {
     
     private let customer: Customer
-    private let purchaseUnits: [PurchaseUnit]
+    private let purchaseUnits = [PurchaseUnit()]
```

Comment:
Nit but I think we generally prefer this style for setting up the empty array:
```suggestion
    private let purchaseUnits: [PurchaseUnit] = []
```

Reply:
ahhh i see, in this case was wanting to initialize and fill the array with the `PurchaseUnit` object inline since it is no longer initialized with the `merchantID` as [before](https://github.com/braintree/braintree_ios/blob/d57b773c54cd95bc3d73786baabe6059ee1cfc46/Sources/BraintreeShopperInsights/BTEligiblePaymentsRequest.swift#L72)

Reply:
Ah, gotcha. We can also consider moving that initialization down to the `init` and keep these constants consistent:
```swift
private let purchaseUnits: [PurchaseUnit]
private let preferences: Preferences
...
init(email: String?, phone: Phone?) {
    self.customer = Customer(email: email, phone: phone)
    self.preferences = Preferences()
    self.purchaseUnits = [PurchaseUnit()]
}
``` 
Up to you though, this isn't a blocker so I'll leave it up to you to decide what's best! 😄

Reply:
i like that! updated [here](465e89dfc0a47eb4fe796ac3f2afcb943ccd2edd)